projects/ion/src/lib/divider/divider.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+
+type Orientation = 'horizontal' | 'vertical';
+
+type Type = 'solid' | 'dashed' | 'text';
+
+export interface IonDividerProps {
+  label?: string;
+  orientation?: Orientation;
+  type?: Type;
+}
+
+@Component({
+  selector: 'ion-divider',
+  templateUrl: './divider.component.html',
+  styleUrls: ['./divider.component.scss'],
+})
+export class DividerComponent {
```

Comment:
```suggestion
export class IonDividerComponent {
```
We need to talk about this pattern, some components have 'Ion' as prefix, some doesn't.